pages/components/List/list.css

```diff
@@ -0,0 +1,32 @@
+.list-group {
+  display: flex;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  padding-left: 0;
+  margin-bottom: 0;
+  border-radius: 0.25rem;
+}
+
+.list-group-item {
+  position: relative;
+  display: block;
+  padding: 0.5rem 1rem;
```

Comment:
very small values(<1rem) can be changed to px

Reply:
Ack, done.